WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingView.swift

```diff
@@ -79,7 +83,14 @@ struct BlazeAdDestinationSettingView: View {
                     Button(Localization.cancel) {
                         dismiss()
                     }
-                    .foregroundColor(Color(uiColor: .accent))
+                }
+
+                ToolbarItem(placement: .confirmationAction) {
+                    Button(Localization.save) {
+                        viewModel.confirmSave()
+                        onDismiss()
```

Comment:
You can use the `dismiss` method to dismiss the view. I don't think you need the `onDismiss` closure is necessary. 
```suggestion
                        dismiss()
```